### ISSUE TITLE
Drop the data- prefix on the slide background attributes

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-reveal.js#18]
Reveal.js recognizes a handful of data attributes (attributes with the prefix `data-`) for setting the background properties of a slide. Currently, the Asciidoctor section template passes these attributes through verbatim. I don't think we need the `data-` prefix and therefore propose to drop it.

The `data-` prefix is required in the HTML as a namespace. We don't need this prefix in the AsciiDoc document as it doesn't add any semantic value nor is is required for compliance.

Before

```
[data-background-image=evil-minion.png]
== Lab results
```

After

```
[background-image=evil-minion.png]
== Lab results
```

For backwards compatibility, we could allow both, but we should definitely only document the attribute names without the `data-` prefix.

**Comment by obilodeau** [comment]
+1

**Comment by obilodeau** [comment]
This will be fixed by PR #52 which will get merged once reviewed. Closing.
